client/my-sites/site-settings/jetpack-site-stats.jsx

```diff
@@ -171,6 +172,10 @@ class JetpackSiteStats extends Component {
 						}
 					</FormFieldset>
 				</FoldableCard>
+
+				<CompactCard href={ '/stats/day/' + siteId }>
```

Comment:
> Should insights be the default view instead?

Not anymore - since #10567 the `day` is the default view.

> There's a utility that figures out where should the My Sites nav item send you to.

Good call, I've updated this PR to use the same utility that's used there in 73a046457647b860120cd4fbfe9c9a8fc5d5e205.